ffcx/codegeneration/C/c_implementation.py

```diff
@@ -156,6 +156,8 @@ def _dtype_to_name(self, dtype) -> str:
             return dtype_to_c_type(self.real_type)
         if dtype == L.DataType.INT:
             return "int"
+        if dtype == L.DataType.BOOL:
+            return "bool"
```

Comment:
I am not sure about using `bool` in C? Is it defined? Maybe in C99, but I think then you are supposed to include `stdbool.h`?

Reply:
https://github.com/FEniCS/ffcx/blob/main/ffcx/codegeneration/ufcx.h#L26